src/flow.ts

```diff
@@ -216,6 +216,7 @@ export class Flow {
     // Convert the array of font names into an array of Font objects.
     const fonts = fontNames.map((fontName) => Font.load(fontName));
     Tables.MUSIC_FONT_STACK = fonts;
+    Glyph.MUSIC_FONT_STACK = fonts.slice();
```

Comment:
This is so that the Glyph class does not need to reference `Tables.MUSIC_FONT_STACK`. Instead, the font stack is set every time someone calls Flow.setMusicFont(...).

Reply:
Ok